analytic_engine/src/sst/meta_data/cache.rs

```diff
@@ -22,8 +22,8 @@ pub type MetaCacheRef = Arc<MetaCache>;
 pub struct MetaData {
     /// The extended information in the parquet is removed for less memory
     /// consumption.
-    parquet: parquet_ext::ParquetMetaDataRef,
-    custom: ParquetMetaDataRef,
+    pub parquet: parquet_ext::ParquetMetaDataRef,
+    pub custom: ParquetMetaDataRef,
```

Comment:
I guess two methods called `parquet()` and `custom` has provided read-only access already.